apps/ensindexer/.env.local.example

```
@@ -22,11 +22,24 @@
 # The following environment variables are supported:
 # - ALCHEMY_API_KEY — if set, Alchemy RPC URLs (HTTP & WS) will be provided for each of the chains it supports
 # - DRPC_API_KEY — if set, an DRPC RPC URL (HTTP) will be provided for each of the chains it supports
+# - QUICKNODE_API_KEY & QUICKNODE_ENDPOINT_NAME — if both set, an QuickNode RPC URL will be provided
```

Please make it explicit what happens if only one of these is set (not both). A quick suggestion is to reject the config and terminate as it's probably a situation where it's better to fail loud to help the operator avoid a situation where they accidentally made a mistake.